consensus/propagation/commitment_state.go

```diff
@@ -134,30 +128,20 @@ func (p *ProposalCache) unfinishedHeights() []*proposalData {
 // example, passing the height that was already committed is not actionable.
 // Passing a round that has already been surpassed is not actionable.
 func (p *ProposalCache) relevant(height int64, round int32) bool {
-	if height < p.consensusHeight {
-		return false
-	}
-
-	if round < p.consensusRound {
+	// TODO Ping @evan-forbes whether this should be:
```

Comment:
@evan-forbes I think this should be:
```
p.height != height || (p.height == height && p.round != round)
```

Because we will never accept any future proposal. So, we can restrain this check instead of using `>`